wrappers/winbase.go

```diff
@@ -140,6 +140,16 @@ const (
 	CREATE_NO_WINDOW                 = 0x08000000
 )
 
+const (
+	VOLUME_NAME_DOS  = 0x0
+	VOLUME_NAME_GUID = 0x1
+	VOLUME_NAME_NT   = 0x2
+	VOLUME_NAME_NONE = 0x4
+
+	FILE_NAME_NORMALIZED = 0x0
+	FILE_NAME_OPENED     = 0x8
+)
```

Comment:
Let's split these into two `const` blocks, one for the `VOLUME_NAME_` constants and one for the `FILE_NAME_` constants.